Library/Homebrew/utils.rb

```diff
@@ -286,12 +315,7 @@ def puts_columns(items, highlight = [])
   rows = (items.size + cols - 1) / cols
   cols = (items.size + rows - 1) / rows # avoid empty trailing columns
 
-  plain_item_lengths = items.map(&:length) if cols >= 2
-  if highlight && highlight.any?
-    items = items.map do |item|
-      highlight.include?(item) ? "#{Tty.highlight}#{item}#{Tty.reset}" : item
-    end
-  end
+  plain_item_lengths = items.map(&method(:output_length)) if cols >= 2
```

Comment:
Is there any benefit over `items.map { |s| output_length(s) }` that, to me, seems easier to grasp?